spec/lib/schedule_spec.rb

```diff
@@ -44,26 +45,21 @@
       # 24<25>26 27 28 29 30
       # 31
 
-      start_date = Date.new(2014, 8, 18) # Monday
-      expect(subject.dates(start_date).first).to eq(start_date + 4) # Friday
-
-      start_date = Date.new(2014, 8, 19) # Tuesday
-      expect(subject.dates(start_date).first).to eq(start_date + 4) # Saturday
-
-      start_date = Date.new(2014, 8, 20) # Wednesday
-      expect(subject.dates(start_date).first).to eq(start_date + 7) # Wednesday
-
-      start_date = Date.new(2014, 8, 21) # Thursday
-      expect(subject.dates(start_date).first).to eq(start_date + 7) # Thursday
-
-      start_date = Date.new(2014, 8, 22) # Friday
-      expect(subject.dates(start_date).first).to eq(start_date + 7) # Friday
-
-      start_date = Date.new(2014, 8, 23) # Saturday
-      expect(subject.dates(start_date).first).to eq(start_date + 6) # Friday
-
-      start_date = Date.new(2014, 8, 24) # Sunday
-      expect(subject.dates(start_date).first).to eq(start_date + 5) # Friday
+      week_start = monday = Date.new(2014, 8, 18)
+      first_bookable_visits = {
+        :monday => week_start.next_day(Date::DAYS_INTO_WEEK[:friday]),
+        :tuesday => week_start.next_day(Date::DAYS_INTO_WEEK[:saturday]),
+        :wednesday => week_start.next_week(:wednesday),
```

Comment:
Durham have visits every day. When it's Wednesday there are 4 whole working days before next Wednesday. Should the next possible bookable day be Tuesday?